### PR TITLE
SAR-4183 Controller only for not eligible page

### PR DESCRIPTION
**Pull request by Daveo-G**

### CODE REVIEW

File: app/incometax/eligibility/controllers/NotEligibleForIncomeTaxController.scala [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package incometax.eligibility.controllers
+
+import core.audit.Logging
```
Comment (kabuumu) [inline app/incometax/eligibility/controllers/NotEligibleForIncomeTaxController.scala]: Unused imports

File: app/incometax/eligibility/controllers/NotEligibleForIncomeTaxController.scala [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+@Singleton
+class NotEligibleForIncomeTaxController @Inject()(val baseConfig: BaseControllerConfig,
+                                                  val messagesApi: MessagesApi,
+                                                  val authService: AuthService
+                                      ) extends StatelessController {
+
+  val show: Action[AnyContent] = Authenticated.asyncUnrestricted { implicit request =>
+    implicit user =>
+      Future.successful(Ok(incometax.eligibility.views.html.not_eligible_for_income_tax(
+      )))
```
Comment (kabuumu) [inline app/incometax/eligibility/controllers/NotEligibleForIncomeTaxController.scala]: Could this just be one line?

File: app/incometax/eligibility/controllers/NotEligibleForIncomeTaxController.scala (hunk 1 above)
Comment (Daveo-G) [reply app/incometax/eligibility/controllers/NotEligibleForIncomeTaxController.scala]: Done

File: app/incometax/eligibility/controllers/NotEligibleForIncomeTaxController.scala (hunk 2 above)
Comment (Daveo-G) [reply app/incometax/eligibility/controllers/NotEligibleForIncomeTaxController.scala]: Done

**Pull request merged**